api/server/server.go

```diff
@@ -41,20 +45,24 @@ func (s *Server) Run() error {
 	router.Use(middleware.Throttle(runtime.NumCPU() * 4))
 	router.Use(middleware.Compress(5, "application/json"))
 	router.Use(middlewareLogger(s.logger))
+	router.Use(middlewareNodeVersion)
 
 	router.Get("/v1/node/identity", api.Handler(s.node.Identity))
 	router.Get("/v1/node/peers", api.Handler(s.node.Peers))
 	router.Get("/v1/node/topics", api.Handler(s.node.Topics))
 	router.Get("/v1/node/health", api.Handler(s.node.Health))
 	router.Get("/v1/validators", api.Handler(s.validators.List))
+	router.Get("/v1/exporter/decideds", api.Handler(s.exporter.Decideds))
+	router.Post("/v1/exporter/decideds", api.Handler(s.exporter.Decideds))
```

Comment:
why do we need `Post`?

Reply:
This is intended to avoid breaking changes at this stage.

Reply:
@olegshmuelov can you please add a comment explaining this?